history/4499.md

```diff
@@ -0,0 +1 @@
+* BMurri: WIXBUG:4499 - Bind MBApreq language to correct langid
```

Comment:
Despite the name, GetUserDefaultLangID gets the language of the installed OS, not of the user. It has never gotten the "user" language as far as I've been able to determine. http://blogs.msdn.com/b/heaths/archive/2005/02/17/375830.aspx.

Please tell me naming and descriptions that are more useful without being incorrect.